core/dbt/task/runnable.py

```diff
@@ -128,7 +149,7 @@ def call_runner(self, runner):
             with startctx, extended_metadata:
                 logger.debug('Began running node {}'.format(
                     runner.node.unique_id))
-            status = 'error'  # we must have an error if we don't see this
+            status: Dict[str, str] = {'state': 'error'}
```

Comment:
It looks to mke like `BaseRunner.get_result_status` returns a dict with a key named `node_status`, not `state`. It looks to me like setting the status to `'error'` as we previously did was outright incorrect, but do we not want the key to be `node_status` here instead?

Reply:
You're right - `state` is something else!